released/packages/coq-dblib/coq-dblib.8.6.0/opam

```diff
@@ -0,0 +1,14 @@
+opam-version: "1.2"
+maintainer: "matej.kosik@inria.fr"
```

Comment:
My suggestion was to change just the `maintainer` field in the `opam` file. The other two files (`descr`, `url`) can be kept verbatim.

That would be, I think, consistent with the meaning of the "maintainer" field (in OPAM-speak).

In other words, if:
- you have created this OPAM package (even if you copied files)
- you have modified your copy of those files (as you did)
- you have run `opam lint`
- you have tested whether this new OPAM package is installable
- you have tested whether this new OPAM package is uninstallable
- you are tracking installability of this OPAM package as new Coq versions are rolled out (satisfying the constraint you specified)
- you want to be contacted in case some user has troubles installing the package you have created

then you should be the maintainer. I do not think it is honest for me to claim that I did/do those things.

For `coq-aac-tactics`, the situation is similar. If you have marked me as a maintainer, I'll be happy to merge a pull-request where you update this information so that you are declared as the maintainer (consistently with OPAM-speak --- e.g. in Debian-speak it you would be "packager").